presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/CommandExecutor.java

```diff
@@ -112,10 +107,12 @@ private String getProcessOutput(Process process)
         catch (InterruptedException e) {
             LOGGER.error(e, "Process failed to complete.");
         }
+        // TODO: Should this throw an exception instead of returning null?
+        // How will the caller handle this null?
```

Comment:
I've removed that comment. But should I replace it with an issue, or should I apply some sort of fix in another pull request?

(I don't really have time for more pull requests, though...)